metabase/metabase
# Mantine Button

Changes:
- Exports `Button` and add style overrides
- Adds focus ring styles

How to verify:
- Run `yarn storybook`
- Open http://localhost:6006/?path=/story/buttons-button--default-story
- Make sure it works according to the figma file https://www.figma.com/file/Ey1rOyIxRHpmRvE9XrGyop/Buttons-%2F-Button?type=design&node-id=1-96&mode=design&t=DaIKPKduzsl5PHpd-0
- For focus ring, click somewhere in "canvas" and press `Tab`

<img width="1015" alt="Screenshot 2023-08-16 at 15 33 06" src="https://github.com/metabase/metabase/assets/8542534/d72fc38e-c973-4753-8d34-ffa1e4e64a7a">
